docs/accessibility-statement.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Accessibility Statement
+---
+
+_Last updated: April 7, 2019_
+
+We want everyone who uses Gatsby to feel welcome and find the experience rewarding: this includes developers with disabilities and users with disabilities for Gatsby sites, docs, and resources.
+
+This page was created to collect accessibility information about the Gatsby ecosystem in one place and provide communication channels for people with disabilities to get help with Gatsby.
+
+## Gatsby products and services
+
+Gatsbyjs.org is the online home of the open source Gatsby website framework; it includes documentation, tutorials, and guides on how and why to build with Gatsby, as well as a blog, site showcase, and information on how to contribute to the project.
+
+Gatsbyjs.com is the business website for Gatsby, Inc. the startup building Gatsby, including our first product, Gatsby Preview: a cloud service allowing team members to collaborate on a Gatsby site in development.
+
+## Providing feedback and getting accessibility help
+
+We strive to make these websites and the Gatsby framework itself as accessible as possible. Our goal is to meet WCAG 2.0 AA, with which we are partially compliant. We’ll be the first to admit the Gatsby ecosystem is a work in progress, and we are open to all feedback to make things better.
+
+To contact the core team with your accessibility feedback or challenges, please [file an issue on Github](https://github.com/gatsbyjs/gatsby/issues/new/choose).
+
+Alternatively, we welcome you to reach out directly to Marcy Sutton, Head of Learning at Gatsby: [marcy@gatsbyjs.com](mailto:marcy@gatsbyjs.com)
+
+## Building with Gatsby
+
+To learn how to build an accessible website with Gatsby, visit our [docs accessibility page](/docs/making-your-site-accessible/). Contributions are very welcome as this page evolves.
+
+## Third-party platforms, products and services
+
+Common Gatsby workflows involve third-party products and services, such as sourcing from Content Management Systems, and managing and deploying sites with hosting platforms. Some of these platforms appear to have been designed and/or built without accessibility in mind; we encourage those vendors to improve their accessibility along with us.
+
+Our current recommendations for accessible CMS platforms are:
+
+- Wordpress with the Classic Editor Plugin
+- Drupal
+
+For managing and deploying Gatsby sites, we recommend:
```

Comment:
Is this based on accessibility reasons? If so, may want to mention that too, e.g.

```suggestion
For managing and deploying Gatsby sites with accessible tools, we recommend:
```